UI/MemStickScreen.cpp

```diff
@@ -116,7 +117,7 @@ MemStickScreen::MemStickScreen(bool initialSetup)
 		if (System_GetPropertyBool(SYSPROP_ANDROID_SCOPED_STORAGE)) {
 			choice_ = CHOICE_BROWSE_FOLDER;
 		} else {
-			WARN_LOG(SYSTEM, "Scoped storage not enabled - shouldn't be in MemStickScreen at initial setup");
+			WARN_LOG_REPORT(SYSTEM, "Scoped storage not enabled - shouldn't be in MemStickScreen at initial setup");
```

Comment:
During initialsetup reporting will never be enabled so I don't think this can even happen. But yeah, I guess if you somehow end up in initialsetup a second time, we'd like to know...